go-controller/pkg/ovn/master.go

```diff
@@ -471,6 +471,31 @@ func (oc *Controller) SetupMaster(masterNodeName string) error {
 	return nil
 }
 
+func addNodeLogicalSwitchPort(logicalSwitch, portName, portType, addresses, options string) (string, error) {
+	stdout, stderr, err := util.RunOVNNbctl("--", "--may-exist", "lsp-add", logicalSwitch, portName,
+		"--", "lsp-set-type", portName, portType,
```

Comment:
I'm guessing this is OK, since the tests passed...but just to make sure: is it OK that you always include these commands even when their value is empty? For example, setting port type to ""? Doesn't cause any errors?

Reply:
@trozet, it should be fine.  Port type "" is the default and means [the LSP is a "regular" VIF](https://github.com/ovn-org/ovn/blob/ecc7bd74d8c045c15ea27954f5441d8d21198d50/ovn-nb.xml#L616).

For `lsp-set-options` too, passing an empty string is fine as it will generate an empty map.